### PR TITLE
Connect room name overflow

### PR DESCRIPTION
**Pull request by Veggier**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description
change the tag to h2 to get enough room to avoid the overflow, reference #9499
## QA Instructions, Screenshots, Recordings
image

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by rhymes** [comment]
Hi @Veggier, can you rebase it on top of the latest master? I believe the fix of the flaky tests lies within https://github.com/forem/forem/pull/9556 - let me know if you'd rather be one of us to do the rebase :)

File: app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx
Comment (ludwiczakpawel) [inline app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx:11]: I'm not sure if replacing H1 with H2 is good in terms of accessibility - I'm not saying we use H* tags correctly but this feels more like workaround and not real fix. 

I'd much more recommend updating the style of H1 tag to make it smaller if that's the idea.

File: app/assets/stylesheets/chat.scss
Comment (ludwiczakpawel) [inline app/assets/stylesheets/chat.scss:11]: Would you mind explaining this a little? It's 10px diff so I wonder what's exactly going on there :)

File: app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx
Comment (Veggier) [reply app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx:11]: I will try to change it.

File: app/assets/stylesheets/chat.scss
Comment (Veggier) [reply app/assets/stylesheets/chat.scss:11]: I feel the left sidebar width was too much for the iPad size, so I reduce 10 px.

**Comment by Veggier** [comment]
> Hi @Veggier, can you rebase it on top of the latest master? I believe the fix of the flaky tests lies within #9556 - let me know if you'd rather it'd be one of us to do the rebase :)
> 
> Thank you!

I will try to the rebase on my own first.

File: app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx
Comment (Veggier) [reply app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx:11]: Also, I think we need to limit the length of `channelName`

Comment (Veggier) [reply app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx:11]: I will set the default tag as h2, and if the screen width is more than 1300px, then set the h2 tag bigger which same as h1 tag.

**Comment by Veggier** [comment]
> Hi @Veggier, can you rebase it on top of the latest master? I believe the fix of the flaky tests lies within #9556 - let me know if you'd rather it'd be one of us to do the rebase :)
> 
> Thank you!

I did the rebase, and the job `19667.2` fail for this time

**Comment by rhymes** [comment]
@Veggier restarted it, another flaky test :-|

File: app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx
Comment (benhalpern) [reply app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx:11]: @ludwiczakpawel ping to circle back here?

**Comment by msarit** [comment]
@ludwiczakpawel Hey Pawel 👋🏾  Just drawing your attention back to the changes you requested 🙏🏾

File: app/assets/stylesheets/chat.scss
Comment (ludwiczakpawel) [reply app/assets/stylesheets/chat.scss:11]: Hey @Veggier, can we keep it as it was on 260px? We have some tabs and icons there that would wrap to another line if we reduce it to 250px.

| 260px | 250px |
|---|---|
| image | image |

Especially it's not a big difference visually so I don't think it's really necessary.

**Comment by ludwiczakpawel** [comment]
hey @Veggier it's been a while, sorry for that - this kinda slipped my attention so I apologize for not circling back earlier.

I've noticed one UI regression that will come with this PR so if you could take a look at it whenever you have time, it would be great!

https://github.com/forem/forem/pull/9561#discussion_r550037880

Otherwise, it's looking good.

**Review by ludwiczakpawel — CHANGES_REQUESTED** [review]
hey @Veggier it's been a while, sorry for that - this kinda slipped my attention so I apologize for not circling back earlier.

I've noticed one UI regression that will come with this PR so if you could take a look at it whenever you have time, it would be great!

https://github.com/forem/forem/pull/9561#discussion_r550037880

Otherwise, it's looking good.

**Comment by msarit** [comment]
Hey @Veggier! 👋🏾 
Care to take another look at this PR? You have some conflicts, and you received some feedback as well.
Let us know if you need any help or have questions. 🙏🏾

**Comment by aitchiss** [comment]
Hi @Veggier - just checking in with you about this PR 🙂

Are you still keen to continue work on this? Let us know if you need any assistance with the feedback received so far, or help resolving conflicts.

File: app/assets/stylesheets/chat.scss
Comment (citizen428) [reply app/assets/stylesheets/chat.scss]: @ludwiczakpawel I changed it back now.

**Comment by citizen428** [comment]
Since there was no answer from the original PR author for a while I now resolved the conflicts myself and made the change back from `250px` to `260px` as requested by @ludwiczakpawel. Please re-review, thanks!

**Review by ludwiczakpawel — APPROVED** [review]

**Review by msarit — APPROVED** [review]
LGTM!

**Comment by msarit** [comment]
@citizen428 Approved! Leaving this note here for you since there is no `merge-by-any-core` label on the PR.

**Comment by citizen428** [comment]
> Approved! Leaving this note here for you since there is no merge-by-any-core label on the PR.

@msarit This only applies to team internal PRs. Community PRs can always be merged once they have the necessary approvals.

**Pull request merged**
